.gitmodules

```diff
@@ -3,7 +3,7 @@
 	url = https://github.com/ARMmbed/mbedtls
 [submodule "lib/chillbuff"]
 	path = lib/chillbuff
-	url = https://github.com/GlitchedPolygons/chillbuff
+	url = https://github.com/rhythmize/chillbuff
```

Comment:
I had to make changes in chillbuff as well. I think it'll be better to first merge the chillbuff PR, if you don't have any concerns. And then I'll change the remote back to your repo for chillbuff

Reply:
I merged your chillbuff PR, you can revert the .gitmodules mod. and I'll look into this PR here now :)

Reply:
Reverted